evals/gaia/gaia.py

```diff
@@ -0,0 +1,93 @@
+from huggingface_hub import snapshot_download
+from pathlib import Path
+from inspect_ai import task
+from inspect_ai.scorer import Score, match
+from inspect_ai.solver import basic_agent, system_message, TaskState
+from inspect_ai.tool import web_search, bash
+from inspect_ai.scorer import scorer, Target
+from inspect_ai.dataset import Sample, hf_dataset
+from typing import Callable, Literal
+from inspect_ai import Task
+from inspect_ai.dataset._dataset import FieldSpec
+import os
+from typing import Sequence
+
+# TODO: The constants actaully make thins a bit uglier, I'd love it if people were like "wow look at how clean it is to onboard the dataset"
+DEFAULT_INPUT_PROMPT = """Please answer the question below. You should:
+
+- Return only your answer, which should be a number, or a short phrase with as few words as possible, or a comma separated list of numbers and/or strings.
+- If the answer is a number, return only the number without any units unless specified otherwise.
+- If the answer is a string, don't include articles, and don't use abbreviations (e.g. for states).
+- If the answer is a comma separated list, apply the above rules to each element in the list.
+
+Any files or attachments mentioned in the question can be found in the /shared_files/ directory (some questions do not have associated files). Here is the question:
```

Comment:
Ideally I would tell the agent "in your home directory", but I can't with sample init. I think this is probably fine, but noting that.

Reply:
hmm maybe I'm wrong, but isn't the user behind bash tools etc. defined by the tools themselves? So when the agent does something like "ls ~" we don't know what user it might be?

Reply:
I am not sure I am following. If we just want to make sure that the files are in the home directory we can just make sure that's the default working_dir for the container. I've seen a of models just do "ls ." (rather than "ls ~") so maybe just saying "it's in the working directory" is better? Again, not fully following what the problem/solution is, maybe better discussed realtime...

Reply:
As in, my understanding of how the tools work is that the creator of the tools defines which user the tools execute as, by passing the "user" parameter when doing exec() calls into the container. Therefore,  what the agent sees as "your home directory" is actually defined by the tools? (bash is just one example, there could be others)

Reply:
But for the ordinary case the tools just execute as "the default user for the container". I think a tool that decided to use a different would be very much a bespoke thing for a particular dataset not something we need to deal with in the general case. Let's just try to wrestle this one down realtime tomorrow.

Reply:
Ok makes sense! I think maybe it actually just makes sense to keep it as "shared files" for now, since I now remember models getting confused with their own home directory.